README.md

```diff
@@ -728,9 +728,7 @@ in the Migration dashboard.
 
 ## Jobs Static Code Analysis Workflow
 
-> Please note that this is an experimental workflow.
-
-The `experimental-workflow-linter` workflow lints accessible code from 2 sources:
+The `workflow-linter` workflow lints accessible code from 2 sources:
```

Comment:
just remove it and modify the assessment documentation. it's part of assessment and migration progress

Reply:
done